guides/user/pipeline/triggers/index.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: single
+title:  "Overview"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+A pipeline trigger defines when to automatically run a pipeline. There are a
```

Comment:
s/are/is/, grammatically speaking. But that sounds a little dissonant with "...triggers." 
I suggest "There are many types of triggers available:"